src/content/core/components/namespaces.mdx

```diff
@@ -23,6 +23,21 @@ Switch your kubectl configuration to point to your namespaces with the `okteto k
 
 Once you're done, you can also delete your namespace via the UI, or with the `okteto namespace delete` command.
 
+You can sleep your namespace from the command line with `okteto namespace sleep [name]`.
+
+When you want to wake your namespace, you can use the command `okteto namespace wake [name]`.
+
+You can sleep your namespace from the Okteto UI by clicking on the Sleep dialog.
+
+In order to wake the namespace, you can click on the Wake dialog.
```

Comment:
Would it be worth having a section on awake and sleep and explaining what each is, and including the information here (plus the other ways to awake & sleep). I know this is out of the scope of the change, so feel free to ignore or postpone.

Reply:
Yes, agreed. I think the whole namespace management section of this page needs an overhaul. 
Taking note of this and will follow up in a separate PR